code/modules/hippie/sound/pooling.dm

```diff
@@ -0,0 +1,145 @@
+#define POOL_HIT_COUNT 1
+#define POOL_MISS_COUNT 2
+#define POOLINGS 3
+#define UNPOOLINGS 4
+#define EVICTIONS 5
+#define DEFAULT_POOL_SIZE 50
+
+#ifdef DETAILED_POOL_STATS
+var/global/list/pool_stats = list()
+
+proc/increment_pool_stats(var/type, var/index)
```

Comment:
So basically, you need to get rid of all these procs we will never need, nor use